GooglePayment/src/test/java/com/braintreepayments/api/GooglePaymentUnitTest.java

```diff
@@ -49,6 +66,37 @@ public void setup() {
                     .setTotalPriceStatus(WalletConstants.TOTAL_PRICE_STATUS_FINAL)
                     .setCurrencyCode("USD")
                     .build());
+
+       GoogleApiAvailability mockGoogleApiAvailability = mock(GoogleApiAvailability.class);
+       when(mockGoogleApiAvailability.isGooglePlayServicesAvailable(any(Context.class))).thenReturn(ConnectionResult.SUCCESS);
+
+       mockStatic(GoogleApiAvailability.class);
+       when(GoogleApiAvailability.getInstance()).thenReturn(mockGoogleApiAvailability);
+
+       ActivityInfo mockActivityInfo = mock(ActivityInfo.class);
+       when(mockActivityInfo.getThemeResource()).thenReturn(2132083045);
+
+       mockStatic(ManifestValidator.class);
+       when(ManifestValidator.getActivityInfo(any(Context.class), any(Class.class))).thenReturn(mockActivityInfo);
+    }
+
+    @Test
+    public void requestPayment_whenMerchantNotConfigured_returnsExceptionToFragment() {
+        String configuration = new TestConfigurationBuilder()
+                .paypal(new TestConfigurationBuilder.TestPayPalConfigurationBuilder(true))
```

Comment:
Can we add `.enabled(false)` to this `TestPayPalConfigurationBuilder` to make the test even more readable?

Reply:
the `true` passed into the configuration builder is what gets set to enabled, so adding it would be redundant.  Though, now that you mention it, we don't need to even set PayPal here.  Just building a config works perfectly fine.